app/src/Utils/ProgressBarColours.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * @file This file contains the functions that are used to change the colour of the progress bar
+ */
+
+// type for the colours of the progress bar
+type ProgressBarColors = {
+  box1: string
+  box2: string
+  box3: string
+  bgColor: string
+}
+
+/**
+ *
+ * @param currentBox
+ * @param bgColor
+ * @returns  {ProgressBarColors} - returns the colours for the progress bar
+ */
+
+export const getProgressBarColors = (
+  currentBox: number,
+  bgColor: string,
+): ProgressBarColors => {
+  const boxes = ['#F1F1F1', '#F1F1F1', '#F1F1F1'] // default colour for all boxes while the user is on the same screen
```

Comment:
Please can this colour be defined in StaTheme and then called from there, rather than using hex values anywhere in this file

Reply:
@joefhall good point - will do!

Reply:
I have updated these to use the StaTheme file colours.